### PR TITLE
e2e for mobile data sync

### LINKED ISSUE #8973: Mobile data syncing pop-up missing text
# Problem

On the nightly, the 'sync mobile data' modal shows only the icons for each option, with no text:

## Expected behavior

Text describes each option:

`Continue syncing`
`Stop syncing`

### PR DESCRIPTION
**Pull request by churik**
Added 3 tests about mobile sync after fix #8973

### CODE REVIEW

**Review by Serhy — APPROVED** [review]

File: test/appium/tests/atomic/account_management/test_profile.py [hunk 1]
```diff
@@ -33,20 +33,80 @@ def test_mobile_data_usage_popup_continue_syncing(self):
...
+                self.errors.append("Toggles in Mobile settings are not enabled")
+
+        sign_in_view.just_fyi("Check that can join public chat ans send message")
+        chat_name = sign_in_view.get_public_chat_name()
+        home = profile_view.get_back_to_home_view()
+        chat = home.join_public_chat(chat_name)
+        message = 'test message'
+        chat.chat_message_input.send_keys(message)
+        if not chat.chat_element_by_text(message).is_element_displayed():
+            self.errors.append("Message was not sent!")
```
Comment (Serhy) [inline test/appium/tests/atomic/account_management/test_profile.py:56]: I'd check here the message status after it sent rather than message itself

**Review by Serhy — APPROVED** [review]

File: test/appium/tests/atomic/account_management/test_profile.py (hunk 1 above)
Comment (churik) [reply test/appium/tests/atomic/account_management/test_profile.py:56]: It makes more sense on IOS, on Android you can't even send a message from offline. 
But I can add check

**Comment by Serhy** [comment]
Well done, @churik!

File: test/appium/tests/atomic/account_management/test_profile.py [hunk 2]
```diff
@@ -28,15 +28,86 @@ def test_set_profile_picture(self):
...
+        if sign_in_view.element_by_text("Start syncing").is_element_displayed(20):
+            self.errors.append("Popup is shown, but 'ask me when on mobile network' is disabled")
+
+        sign_in_view.just_fyi("Check 'Restore default' setting")
+        profile_view.element_by_text('Restore Defaults').click()
+        if profile_view.use_mobile_data.attribute_value("checked"):
+            self.errors.append("Mobile data is enabled by default")
+        if not profile_view.ask_me_when_on_mobile_network.attribute_value("checked"):
+            self.errors.append("'Ask me when on mobile network' is not enabled by default")
+
```
Comment (antdanchenko) [inline test/appium/tests/atomic/account_management/test_profile.py:110]: please, add self.verify_no_errors()

File: test/appium/tests/atomic/account_management/test_profile.py (hunk 1 above)
Comment (antdanchenko) [reply test/appium/tests/atomic/account_management/test_profile.py:56]: also, please add self.verify_no_errors() in the end of the test

**Review by antdanchenko — CHANGES_REQUESTED** [review]
There will be no verification in a test which does `self.errors.append('whatever')` if `self.verify_no_errors()` is not added in the end of this test

File: test/appium/tests/atomic/account_management/test_profile.py (hunk 2 above)
Comment (churik) [reply test/appium/tests/atomic/account_management/test_profile.py:110]: thx, added.

File: test/appium/tests/atomic/account_management/test_profile.py (hunk 1 above)
Comment (churik) [reply test/appium/tests/atomic/account_management/test_profile.py]: thx, added.

**Comment by churik** [comment]
@antdanchenko thanks for stepping in!
completely forgot about it.
Addressed, please take a look

**Review by antdanchenko — APPROVED** [review]

**Pull request merged**